### PR TITLE
Access form is simplified

### LINKED ISSUE #5857: Make Access Form similar to Discount Form
The discount form was simplified recently. Please simplify the access form in the same way.

Compare discount form
Screenshot from 2020-12-01 21-33-12

### PR DESCRIPTION
**Pull request by sayak2k1maruti**
Fixes #5857

#### Short description of what this resolves:
As per instruction I have simplified the acess form like discount form

#### Changes proposed in this pull request:

Screenshot from 2020-12-02 19-14-55

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Review by daretobedifferent18 — COMMENTED** [review]
Pls Take out "Status" and always make the status active

**Review by daretobedifferent18 — CHANGES_REQUESTED** [review]
- [ ] Pre-fill "Number of discounted tickets" with "10"
  - [ ] Take out "Status" and always make the status active
  - [ ] Change "Create an access code" ---> "Create a New Access Code"

**Comment by sayak2k1maruti** [comment]
> * [ ]  Pre-fill "Number of discounted tickets" with "10"
> * [ ]  Take out "Status" and always make the status active
> * [ ]  Change "Create an access code" ---> "Create a New Access Code"

I just fixes these issues.
Screenshot from 2020-12-02 19-14-55

File: app/templates/components/forms/events/view/create-access-code.hbs [hunk 1]
```diff
@@ -11,28 +11,8 @@
...
-    </div>
-    <div class="field">
-      <UiRadio
-        @name="status"
-        @label="Inactive"
-        @value="false"
-        @current={{this.data.isActive}}
-        @onChange={{action (mut this.data.isActive)}} />
-    </div>
+      @value={{10}} />
```
Comment (daretobedifferent18) [inline app/templates/components/forms/events/view/create-access-code.hbs:15]: Don't do it here.

File: app/templates/components/forms/events/view/create-access-code.hbs [hunk 2]
```diff
@@ -11,28 +11,8 @@
     <Input
       @type="number"
       @name="number_of_access_tickets"
-      placeholder="0"
       @min={{0}}
```
Comment (daretobedifferent18) [inline app/templates/components/forms/events/view/create-access-code.hbs:14]: change to ```"0"```

File: app/templates/components/forms/events/view/create-access-code.hbs (hunk 1 above)
Comment (sayak2k1maruti) [reply app/templates/components/forms/events/view/create-access-code.hbs:15]: okay . I am fixing it

File: app/templates/components/forms/events/view/create-access-code.hbs (hunk 2 above)
Comment (daretobedifferent18) [reply app/templates/components/forms/events/view/create-access-code.hbs:14]: And also mention which Issues u r solving in description

Comment (sayak2k1maruti) [reply app/templates/components/forms/events/view/create-access-code.hbs:14]: Should I remove placeholder???

Comment (daretobedifferent18) [reply app/templates/components/forms/events/view/create-access-code.hbs:14]: yes

Comment (daretobedifferent18) [reply app/templates/components/forms/events/view/create-access-code.hbs:14]: Also Pls refer the #5750

Comment (sayak2k1maruti) [reply app/templates/components/forms/events/view/create-access-code.hbs:14]: Thank U I have fixed it

**Review by vsasvipul0605 — DISMISSED** [review]

**Comment by sachinchauhan2889** [comment]
edit first comment. and add fixes #issueNumber.
this will help us to access issue after merging your pr and close that issue.

**Review by daretobedifferent18 — CHANGES_REQUESTED** [review]
Creating new access ccode results in inactive status of code. It should be active by default
Screenshot from 2020-12-03 12-26-46

**Comment by sayak2k1maruti** [comment]
> Creating new access ccode results in inactive status of code. It should be active by default
> Screenshot from 2020-12-03 12-26-46

I have fixed this issue.
Screenshot from 2020-12-03 16-03-40

**Comment by sayak2k1maruti** [comment]
> edit first comment. and add fixes #issueNumber.
> this will help us to access issue after merging your pr and close that issue.

I have just did this. Thank you.

File: app/templates/components/forms/events/view/create-access-code.hbs [hunk 3]
```diff
@@ -60,65 +40,41 @@
...
-        </div>
-      </div>
-      <div class="ui hidden divider"></div>
-      {{t 'Validity Period (Optional)'}}
-      <div class="fields">
-        <div class="wide field {{if this.device.isMobile 'sixteen' 'five'}}">
-          <label>{{t 'Valid from'}}</label>
-          <Widgets::Forms::DatePicker
+      <div class="field">
+          <label>{{t 'Access Valid from'}}</label>
```
Comment (daretobedifferent18) [inline app/templates/components/forms/events/view/create-access-code.hbs:44]: Pls correct the indentation

Comment (sayak2k1maruti) [reply app/templates/components/forms/events/view/create-access-code.hbs:44]: Sorry I can't understand please elaborate

Comment (daretobedifferent18) [reply app/templates/components/forms/events/view/create-access-code.hbs:44]: Indentation refers to the spaces at the beginning of a code line.

Comment (sayak2k1maruti) [reply app/templates/components/forms/events/view/create-access-code.hbs:44]: o sorry I will fix this

Comment (sayak2k1maruti) [reply app/templates/components/forms/events/view/create-access-code.hbs]: I'd  just did it

File: app/templates/components/forms/events/view/create-access-code.hbs [hunk 4]
```diff
@@ -60,65 +40,40 @@
...
             @id="start_time"
             @value={{this.data.validFromTime}}
             @rangePosition="start" />
         </div>
-        <div class="wide field {{if this.device.isMobile 'sixteen' 'five'}}">
-          <label>{{t 'Expires on'}}</label>
+       </div>
+      </div>
+    <div class="field">
+       <label>{{t 'Access Expires on'}}</label>
```
Comment (daretobedifferent18) [inline app/templates/components/forms/events/view/create-access-code.hbs:61]: here also

Comment (sayak2k1maruti) [reply app/templates/components/forms/events/view/create-access-code.hbs:61]: done

**Comment by daretobedifferent18** [comment]
@sayak2k1maruti Pls take care of this fact, whenever u r taking new issues, pls create a new branch and then start working on it.

**Comment by sayak2k1maruti** [comment]
> @sayak2k1maruti Pls take care of this fact, whenever u r taking new issues, pls create a new branch and then start working on it.

My pull request is closed what to do?

**Comment by iamareebjamal** [comment]
> We don't accept pull requests from `development` branch. Please send a pull request from some other branch. Thank You!

**Comment by iamareebjamal** [comment]
```
git checkout -b access-form
git push origin access-form
```

Create a new PR

**Comment by sayak2k1maruti** [comment]
Should I delete my forked repo and start from very beginning or it can be done from existing forked repo?

**Pull request closed without merging**